src/config/config.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require('sequelize')
+const Op = Sequelize.Op
+
+require('dotenv').config()
+
+module.exports = {
+  development: {
+    url: process.env.DATABASE_URL,
+    dialect: 'postgres',
+    operatorsAliases: {
+      $and: Op.and,
+      $or: Op.or,
+      $eq: Op.eq,
```

Comment:
put these all in an object in another file and spread it. this code does not look good.